crypto/verifier.go

```diff
@@ -20,19 +20,31 @@ import (
 	"crypto/rsa"
 	"crypto/x509"
 	"encoding/asn1"
+	"encoding/json"
 	"encoding/pem"
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"math/big"
 
+	"github.com/benlaurie/objecthash/go/objecthash"
 	"github.com/google/trillian/crypto/sigpb"
 )
 
 // ErrVerify occurs whenever signature verification fails.
 var ErrVerify = errors.New("signature verification failed")
 
+// PublicKeyFromFile returns the public key contained in the keyFile.
+func PublicKeyFromFile(keyFile string) (crypto.PublicKey, error) {
+	pemData, err := ioutil.ReadFile(keyFile)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read: %s. %v", keyFile, err)
+	}
+	return PublicKeyFromPEM(pemData)
+}
+
 // PublicKeyFromPEM converts a PEM object into a crypto.PublicKey
-func PublicKeyFromPEM(pemEncodedKey string) (crypto.PublicKey, error) {
+func PublicKeyFromPEM(pemEncodedKey []byte) (crypto.PublicKey, error) {
```

Comment:
Isn't this reversing a review comment I made on the last PR, about PEM parameters having a consistent type?